src/lib/ngsi/StatusCode.cpp

```diff
@@ -124,6 +124,51 @@ std::string StatusCode::render(Format format, const std::string& indent, bool co
 
 /* ****************************************************************************
 *
+* StatusCode::renderV2 -
+*
+* For version 2 of the API, the unnecessary 'reasonPhrase' is removed.
+*/
+std::string StatusCode::renderV2(bool comma)
```

Comment:
Propose a way of provoking an error down there and let's do it ... :-)
Remember 404 Not Found is not an option - it is 200 OK and empty vector now